spec/spec_helper.rb

```diff
@@ -83,3 +83,7 @@ def set_browser_locale(locale)
     request.env["HTTP_ACCEPT_LANGUAGE"] = "#{locale}"
   end
 end
+
+RSpec.configure do |config|
```

Comment:
Pass `&:infer_spec_type_from_file_location!` as an argument to `configure` instead of a block.